scipy/stats/_qmc.py

```diff
@@ -353,6 +400,9 @@ def geometric_discrepancy(
         The distance metric to use. See the documentation
         for `scipy.spatial.distance.pdist` for the available metrics and
         the default.
+    workers : int, optional
+        Number of workers to use for parallel processing. If -1 is given all
+        CPU threads are used. Default is 1.
```

Comment:
Minor: could be worth mentioning that this only applies with `mindist` `method` option (I believe we have seen some credible proposals to add concurrency/acceleration to distance metrics, but that's another matter).

IIRC there was some discussion between Matt and Albert (?) about avoiding kwargs that only apply to a subset of other kwargs, but I don't really see a better option.

Reply:
The alternative would be to rely on Array API support here. Not sure about the plan on the dist part now.

Reply:
OK, I have documented it now.